### ISSUE TITLE
Intermission with bots

### ISSUE THREAD

**Issue by CircaCitadel** [JACoders/OpenJK#844]
Currently, if you have bots on an OpenJK server, if the time limit is reached, the server is stuck on the intermission scoreboard screen until someone comes on to click through it. Is there any way this could be fixed? Perhaps by adding a custom duration for the intermission?

**Comment by ensiform** [comment]
It's not that bots don't ready up.  That's intentional.  It's that there are no humans to start the countdown.  I thought I had merged some fixes for this but I'll look into some options.  It's mod specific though so running ja+ won't do anything etc.

**Comment by CircaCitadel** [comment]
Right. I run my server with just OpenJK. Mods like JA+ already has this fixed I believe.

**Comment by ensiform** [comment]
I don't think JA+ does.  Not in the way I was thinking anyway.

**Comment by deathsythe47** [comment]
Why do bots need to endlessly play each other?

**Comment by CircaCitadel** [comment]
It would be nice to have bots AND a map rotation function together. If I have bots, my server will always be stuck on one map until someone comes on the server.

**Comment by ensiform** [comment]
Because keeping the map on the same map forever isn't good either due to long uptime bugs.  And these bugs aren't fixable in a compatible manner.

**Comment by ensiform** [comment]
I can merge this if wanted: https://github.com/ioquake/ioq3/commit/5e35d88b9bef376b3a163e210b04d16846d8a864#diff-4e895ed59b349afaee5d754f645235ac
